tests/unit/backend/corpora/api_server/base_api_test.py

```diff
@@ -48,7 +53,69 @@ def _remove_timestamps(jrb, removing):
         return _remove_timestamps(body, remove_attributes)
 
 
+def mock_assert_authorized_token(token: str, audience: str = None):
+    if token == "owner":
+        return {"sub": "test_user_id", "email": "fake_user@email.com", "scope": []}
+    elif token == "not_owner":
+        return {"sub": "someone_else", "email": "fake_user@email.com", "scope": []}
+    elif token == "super":
+        return {"sub": "super", "email": "fake_user@email.com", "scope": ["write:collections"]}
+    else:
+        raise Exception()
+
+
+def get_cxguser_token(user="owner"):
+    """
+    Generated an auth token for testing.
+    :param app: a WSGI app.
+    :return:
+    """
+    cxguser = base64.b64encode(
```

Comment:
just curious - why are we encoding in utf8-->b64 and then decoding back to utf8?

Reply:
Since the data portal API uses a different Auth method than the curation portal, we decode the token as part of the authentication process. Right here to be exact https://github.com/chanzuckerberg/single-cell-data-portal/blob/63154b35242213da2d98a596f999a870f7507a0b/backend/corpora/lambdas/api/v1/authentication.py#L225